test/e2e/client_test.go

```diff
@@ -40,21 +44,38 @@ func newTestClient(kubeconfig string) *testClient {
 		panic(err)
 	}
 
+	pc, err := projectclient.NewForConfig(config)
+	if err != nil {
+		panic(err)
+	}
+
 	return &testClient{
 		kc: kc,
+		pc: pc,
 	}
 }
 
-func (t *testClient) createNamespace(namespace string) {
-	// TODO: Create a project request
+func (t *testClient) createNamespace(namespace string) error {
+	if _, err := t.pc.ProjectRequests().Create(&project.ProjectRequest{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: namespace,
+		}}); err != nil {
+		return err
+	}
 	t.namespace = namespace
 	// TODO: Wait for a successful SAR check
+	return nil
```

Comment:
may just be worth putting in some code that waits for the obvious service accounts to be created, before that bites a new starter